src/detect-engine-uint.c

```diff
@@ -65,6 +65,32 @@ int DetectU32Match(const uint32_t parg, const DetectU32Data *du32)
     return 0;
 }
 
+static int DetectU32Validate(DetectU32Data *du32) {
```

Comment:
nit: style (clang format also rejects this)

bigger issue: commit message is inaccurate. We error out on bad rules. No 'poor rule warning' anymore.

Reply:
indeed :-)